addons/mail/static/src/models/messaging_notification_handler.js

```diff
@@ -461,12 +461,26 @@ registerModel({
          * @param {integer[]} param0.messag_ids
          */
         _handleNotificationMessageDelete({ message_ids }) {
+            let decrementInbox = 0;
```

Comment:
Would name those a bit better.
Something like "deletedInboxMessagesCount" or maybe you have better?
Same for the other one.

Reply:
It's a variable in local scope, so not need to `makeALongToReadVariableThatHasAVerySpecificName`.
Short names are easier to read :)

Reply:
Indeed,  deletedInboxMessagesCount is a good name for what it stand for rather than what we want to do with like I have used decrementInbox. So to keep it short, I have removed the count and used a Msg instead of Message which gives: deletedInboxMsg.